artifacts/src/main/resources/issuance/example/credential-offer-message.json

```diff
@@ -0,0 +1,38 @@
+{
+  "@context": [
+    "https://w3id.org/dspace-dcp/v0.8",
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "type": "CredentialOfferMessage",
+  "credentialIssuer": "issuer",
+  "credentials": [
+    {
+      "type": "CredentialObject",
+      "credentialType": [
+        "VerifiableCredential",
+        "CompanyCredential"
+      ],
+      "format": "ldp_vc",
+      "offerReason": "reissue",
+      "bindingMethods": [
+        "did:web"
+      ],
+      "cryptographicSuites": [
+        "JsonWebKey2020"
+      ],
```

Comment:
JsonWebKey2020 is not really a cryptographic suite but a container for holding (public) key material with support for different crytographic suites. Perhaps not an issue for this PR, but might be a new issue

Reply:
Yeah I just copied and pasted the current `CredentialObject` example, we might track this in another issue and fix that there